web/packages/shared/components/AdvancedSearchToggle/AdvancedSearchToggle.tsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2023 Gravitational, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import { Text, Toggle, Link, Flex } from 'design';
+
+import { ToolTipInfo } from 'shared/components/ToolTip';
+
+const GUIDE_URL =
+  'https://goteleport.com/docs/setup/reference/predicate-language/#resource-filtering';
```

Comment:
while we are at it, the URL seemed to have changed to
```suggestion
  'https://goteleport.com/docs/reference/predicate-language/#resource-filtering';
```